src/main/java/frc/robot/subsystems/ArmSystem.java

```diff
@@ -0,0 +1,83 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.*;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class ArmSystem extends SubsystemBase {
+    private final CANSparkMax motor;
+    private final RelativeEncoder neoEncoder;
+    private final DutyCycleEncoder absEncoder;
+    //private final SparkPIDController pidController;
+
+    public ArmSystem() {
+        motor = new CANSparkMax(RobotMap.ARM_MOTOR_PORT, CANSparkLowLevel.MotorType.kBrushless);
+        neoEncoder = motor.getEncoder();
+        absEncoder = new DutyCycleEncoder(RobotMap.ARM_ENCODER_PORT);
+        //pidController = motor.getPIDController();
+
+        motor.restoreFactoryDefaults();
+
+        motor.setInverted(true);
+        /*
+        neoEncoder.setPosition((absEncoder.getAbsolutePosition() - RobotMap.ABSOLUTE_ENCODER_ZERO_OFFSET) * RobotMap.ARM_GEAR_RATIO + Units.Degrees.of(30).in(Units.Revolutions));
```

Comment:
don't keep code in comments, if you ever need it again it is in git stored in an old commit. As such delete this and  all other code in comments.

Reply:
done